src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -0,0 +1,495 @@
+    </div>
+  );
+};
+
+export const ImageOutputList = () => {
+  const distribution = useAppSelector((state) => selectDistribution(state));
+  const arch = useAppSelector((state) => selectArchitecture(state));
+  return (
+    <TextContent>
+      <ReleaseLifecycle />
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Release
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {RELEASES.get(distribution)}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Architecture
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>{arch}</TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+export const FSCList = () => {
+  return (
+    <TextContent>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvAWSList = () => {
+  const { data: rawAWSSources, isSuccess } = useGetSourceListQuery({
+    provider: 'aws',
+  });
+  const awsAccountId = useAppSelector((state) => selectAwsAccountId(state));
+  const awsShareMethod = useAppSelector((state) => selectAwsShareMethod(state));
+
+  const awsSources = extractProvisioningList(rawAWSSources);
+  const { isBeta } = useGetEnvironment();
+
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>AWS</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Shared to account
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {!isBeta() && awsAccountId}
+          {isBeta() && awsShareMethod === 'sources' && isSuccess && (
+            <AwsAccountId />
+          )}
+          {isBeta() && awsShareMethod === 'manual' && awsAccountId}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          {awsShareMethod === 'sources' ? 'Source' : null}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {isSuccess && awsShareMethod === 'sources'
+            ? awsSources?.find((source) => source.id === source)?.name
+            : null}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Default region
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          us-east-1
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvGCPList = () => {
```

Comment:
<img width="657" alt="Screenshot 2024-02-01 at 12 24 07" src="https://github.com/RedHatInsights/image-builder-frontend/assets/73419853/714afc5e-dc9e-4a1d-8078-b62f69fcfe50">
this is how its look in stage, not sure I understand what to change

Reply:
For GCP step we currently have two options: "Share image with a Google acount" and "Share image with Red Hat Insights only". For the "with a Google Account" it makes perfect sense to show an account type and principal, but if we choose the "with Insights only" option, we shouldn't be able to show any account type or principal, because none were chosen. I think in case the image is shared only with insights, we should probably only show something like:

GCP
Image type / Red Hat hosted image
Shared with / Red Hat Insights only